.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -18,3 +18,8 @@ A clear and concise description of any alternative solutions or features you've
 
 **Additional context**
 Add any other context or screenshots about the feature request here.
+
+_____________________________________________________
+**Disclaimer** 
+
+We appreciate your contribution whether it is a bug report, feature request, or pull request with improvement (hopefully). Please comply with the [Community ethics policy](https://opensource.guide/building-community/), and do not expect us to answer your requests immediately. Also, do not treat *GitHub issues* as a support channel.
```

Comment:
The link is referencing a generic web page and not the Community ethics policy

Reply:
Fixed. Changed to a precise Acceptable Use document